src/components/install_assistant/InstallAssistantStep8.vue

```diff
@@ -22,7 +22,8 @@
       </p>
       <p class="font-weight-bold">
         Wichtig: Die "Angaben zur Handhabung von Phasen" des Fahrzeugs
-        ist richtig einzutragen!
+        sind richtig einzutragen! Beachte die Hilfstexte unter den
+        Fragezeichensymbolen.
```

Comment:
```suggestion
        sind richtig einzutragen! Beachte die Hilfstexte beim Klick auf
        die Fragezeichensymbole.
```
Es könnte zur Veranschaulichung auch direkt so ein Symbol dort angezeigt werden.